elastic/integrations
# [Azure OpenAI] Add PTU Deployment Metrics

- Enhancement

- This PR includes the metrics from the PTU deployments of Azure OpenAI services.
- Updated the metric type to long for the total fields after verifying the data in Azure monitor metrics.
- Remove experimental text from the configuration page.


## Checklist




## How to test this PR locally

- Add PTU managed deployments models in the Azure OpenAI service. Install the integration and verify the metrics for PTU managed deployments are getting ingested into elastic.


## Screenshots
